backends/advanced/src/advanced_omi_backend/services/transcription/streaming_consumer.py

```
@@ -164,10 +163,22 @@ async def end_session_stream(self, session_id: str):
+            await self.redis_client.set(completion_key, "1", ex=300)  # 5 min TTL
+            logger.info(f"✅ Streaming transcription complete for {session_id} (signal set)")
 
         except Exception as e:
             logger.error(f"Error ending stream for {session_id}: {e}", exc_info=True)
+            # Still signal completion even on error so conversation job doesn't hang
+            try:
+                completion_key = f"transcription:complete:{session_id}"
+                await self.redis_client.set(completion_key, "error", ex=300)
+                logger.warning(f"⚠️ Set error completion signal for {session_id}")
+            except Exception:
+                pass  # Best effort
```

Ruff (0.14.14)

`try`-`except`-`pass` detected, consider logging the exception

(S110)